main.go

```diff
@@ -221,6 +227,26 @@ func setupControllers(mgr ctrl.Manager, cCache *cache.Cache, queues *queue.Manag
 			os.Exit(1)
 		}
 	}
+	if isFrameworkEnabled(cfg, rayjob.FrameworkName) {
```

Comment:
Before this starts growing too much, maybe we can replace with a loop.

We can have some `RegisterIntegration` method that is called from the `init()` function of each integration's package.

Then the main goes over all the registered integrations, checks if they are enabled before calling the constructor. Otherwise it creates the noop webhook.

Reply:
I did think of something like this but I don't think we should link it to the ray integration, so I've created #691.